composer.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name"       : "automattic/co-authors-plus",
+    "name"       : "automattic/co-authors-plus",
```

Comment:
Formatting looks off here.

Reply:
I noticed that the file was not correctly formatted in general and just adjusted this.

Reply:
Use a `.editorconfig` file, and https://github.com/ergebnis/composer-normalize